.github/ISSUE_TEMPLATE/docs-feedback.yml

```diff
@@ -0,0 +1,47 @@
+name: Learn feedback control.
+description: |
+  ⛔ This template is hooked into the feedback control on the bottom of every page on the learn.microsoft.com site. It automatically fills in several fields for you. Don't use for other purposes. ⛔
+assignees:
+  - BillWagner
```

Comment:
Who else should I add here?

I think I can add a team, as long as the org isn't needed (meaning its a team in the dotnet org). The GitHub docs are unclear.

Reply:
I'd rather not assign a team. It's preferable to have a single owner for each issue.
If you're okay routing/triaging incoming issues by default, that'd be great, but we can discuss some options if the load is too great.

Reply:
Triaging these hasn't been a major time sink in the past. I don't expect that to change, but I'll let everyone know if it is.